ignite/distributed/comp_models/native.py

```diff
@@ -46,40 +46,83 @@ def create_from_context() -> Optional["_NativeDistModel"]:
             return _NativeDistModel()
 
         @staticmethod
-        def create_from_backend(backend: str, **kwargs: Any) -> "_NativeDistModel":
+        def create_from_backend(
+            backend: str,
+            init_method: Optional[str] = None,
+            world_size: Optional[int] = None,
+            rank: Optional[int] = None,
+            **kwargs: Any,
+        ) -> "_NativeDistModel":
             if backend not in _NativeDistModel.available_backends:
                 raise ValueError(f"Backend should be one of '{_NativeDistModel.available_backends}'")
 
             if dist.is_available() and dist.is_initialized():
                 raise RuntimeError("Can not create new distributed process group if default one is already initialized")
-            return _NativeDistModel(backend=backend, **kwargs)
 
-        def __init__(self, backend: Optional[str] = None, timeout: Optional[int] = None, **kwargs: Any) -> None:
+            if init_method is None:
+                if world_size is not None or rank is not None:
+                    raise ValueError("Arguments rank and world_size should be None if no init_method is provided")
```

Comment:
@ahmedo42 there also test to write to increase codecov according to the warnings here:
https://github.com/pytorch/ignite/pull/1767/files